### PR TITLE
Added CuerdOS support

### PR DESCRIPTION
**Pull request by CuerdOS**
It looks really really great

image_2024-05-21_12-17-47

### CODE REVIEW

File: src/detection/os/os_linux.c [hunk 1]
```diff
@@ -157,6 +157,13 @@ static bool detectDebianDerived(FFOSResult* result)
         ffStrbufSetNS(&result->versionID, versionEnd - versionStart, result->prettyName.chars + versionStart);
         return true;
     }
+    else if (ffStrbufStartsWithS(&result->name, "CuerdOS GNU/Linux"))
+    {
+        ffStrbufSetS(&result->name, "CuerdOS GNU/Linux");
+        ffStrbufSetS(&result->id, "cuerdos");
+        ffStrbufSetS(&result->idLike, "debian");
```
Comment (CarterLi) [inline src/detection/os/os_linux.c:164]: Doesn't your os-release file provide these info?

Comment (CuerdOS) [reply src/detection/os/os_linux.c:164]: Yes, this is how we have it set up

imagen

Comment (CarterLi) [reply src/detection/os/os_linux.c:164]: So don't hard code these info. Remove these `ffStrbufSetS`s

Comment (CuerdOS) [reply src/detection/os/os_linux.c]: Did it

**Pull request merged**